options/options.js

```diff
@@ -68,6 +68,20 @@ async function init() {
                 break;
             case "display":
                 updateDisplayElement(optionsElements[i])
+
+                break;
+            case "number-change":
+                let numberChangeOption = optionsElements[i].getAttribute("sync-option");
+                let configValue = SB.config[numberChangeOption];
+                let numberInput = optionsElements[i].querySelector("input");
+
+                if (configValue != undefined) {
```

Comment:
It should never be undefined right? Because of the default.

Also, use `!==` here (I know my code doesn't always do that, but I am doing it in code from now on).